Jenkinsfile

```diff
@@ -163,19 +163,18 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
                 [$class: 'FileBinding', credentialsId: 'opkg-build-private-key', variable: 'PRIVATE_KEY'],
                 [$class: 'FileBinding', credentialsId: 'opkg-build-public-key', variable: 'PUBLIC_KEY'],
             ]){
-                // Attempt to build quickly and reliably
+                sh "cp ${env.PRIVATE_KEY} ${WORKSPACE}/key-build"
+                sh "cp ${env.PUBLIC_KEY} ${WORKSPACE}/key-build.pub"
+
                 try {
```

Comment:
We do not handle any exceptions, but in previous code if the exception contained exit code 143 another error was thrown, and finally block was there to clean up the keys. I imagine we do not want the keys laying around the workspace if we can help it, so although we do not handle any exceptions, we still want to get rid of the keys regardless what happens.